charts/prefect-server/values.yaml

```diff
@@ -10,26 +10,41 @@ commonLabels: {}
 # -- annotations to add to all deployed objects
 commonAnnotations: {}
 
-## Deployment Configuration
-server:
+## Global Deployment Configuration
+global:
```

Comment:
im not exactly sure if this affects anything, but the `global` value is used for making a value available to a chart and subcharts.  i wonder if using this creates any conflicts with the subcharts we have here, like the bitnami or postgres charts (it very well may not)

https://helm.sh/docs/chart_template_guide/subcharts_and_globals/#global-chart-values